src/SampleMods/OWML.LoadCustomAssets/LoadCustomAssets.cs

```diff
@@ -133,28 +134,40 @@ private void OnStartSceneChange(OWScene oldScene, OWScene newScene)
 
 		private void OnCompleteSceneChange(OWScene oldScene, OWScene newScene)
 		{
-			_isInSolarSystem = newScene == OWScene.SolarSystem;
+			bool ready = true;
+			if (Mouse.current == null)
```

Comment:
are these needed?

Reply:
i think you can remove this check and isReady

Reply:
now removed